keyboards/melgeek/z70ultra_v1/info.json

```
@@ -0,0 +1,317 @@
+            { "flags": 4, "matrix": [1, 11], "x": 110, "y": 10 },
+            { "flags": 4, "matrix": [1, 12], "x": 120, "y": 10 },
+            { "flags": 4, "matrix": [1, 13], "x": 130, "y": 10 },
+            { "flags": 1, "matrix": [1, 14], "x": 140, "y": 10 },
+          
+            { "flags": 4, "matrix": [2, 0],  "x": 0,   "y": 20 },
+            { "flags": 4, "matrix": [2, 1],  "x": 10,  "y": 20 },
+            { "flags": 4, "matrix": [2, 2],  "x": 20,  "y": 20 },
+            { "flags": 4, "matrix": [2, 3],  "x": 30,  "y": 20 },
+            { "flags": 4, "matrix": [2, 4],  "x": 40,  "y": 20 },
+            { "flags": 4, "matrix": [2, 5],  "x": 50,  "y": 20 },
+            { "flags": 4, "matrix": [2, 6],  "x": 60,  "y": 20 },
+            { "flags": 4, "matrix": [2, 7],  "x": 70,  "y": 20 },
+            { "flags": 4, "matrix": [2, 8],  "x": 80,  "y": 20 },
+            { "flags": 4, "matrix": [2, 9],  "x": 90,  "y": 20 },
+            { "flags": 4, "matrix": [2, 10], "x": 100, "y": 20 },
+            { "flags": 4, "matrix": [2, 11], "x": 110, "y": 20 },
+            { "flags": 4, "matrix": [2, 13], "x": 130, "y": 20 },
+            { "flags": 4, "matrix": [2, 14], "x": 140, "y": 20 },
+                                       
+            { "flags": 4, "matrix": [3, 0],  "x": 0,   "y": 30 },
+            { "flags": 4, "matrix": [3, 2],  "x": 20,  "y": 30 },
+            { "flags": 4, "matrix": [3, 3],  "x": 30,  "y": 30 },
+            { "flags": 4, "matrix": [3, 4],  "x": 40,  "y": 30 },
+            { "flags": 4, "matrix": [3, 5],  "x": 50,  "y": 30 },
+            { "flags": 4, "matrix": [3, 6],  "x": 60,  "y": 30 },
+            { "flags": 4, "matrix": [3, 7],  "x": 70,  "y": 30 },
+            { "flags": 4, "matrix": [3, 8],  "x": 80,  "y": 30 },
+            { "flags": 4, "matrix": [3, 9],  "x": 90,  "y": 30 },
+            { "flags": 4, "matrix": [3, 10], "x": 100, "y": 30 },
+            { "flags": 4, "matrix": [3, 11], "x": 110, "y": 30 },
+            { "flags": 4, "matrix": [3, 12], "x": 120, "y": 30 },
+            { "flags": 4, "matrix": [3, 13], "x": 130, "y": 30 },
+            { "flags": 4, "matrix": [3, 14], "x": 140, "y": 30 },
+        
+            { "flags": 4, "matrix": [4, 0],  "x": 0,   "y": 40 },
+            { "flags": 4, "matrix": [4, 1],  "x": 10,  "y": 40 },
+            { "flags": 4, "matrix": [4, 2],  "x": 20,  "y": 40 },
+			{ "flags": 4, "matrix": [4, 5],  "x": 40,  "y": 40 },
+            { "flags": 4, "matrix": [4, 5],  "x": 50,  "y": 40 },
+			{ "flags": 4, "matrix": [4, 5],  "x": 60,  "y": 40 },          
+            { "flags": 4, "matrix": [4, 9],  "x": 90,  "y": 40 },
+            { "flags": 4, "matrix": [4, 10], "x": 100, "y": 40 },
+            { "flags": 4, "matrix": [4, 12], "x": 120, "y": 40 },
+            { "flags": 4, "matrix": [4, 13], "x": 130, "y": 40 },
+            { "flags": 4, "matrix": [4, 14], "x": 140, "y": 40 }
+        ]
+    },
```

conforms to PR guidelines
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_ansi_split_space"
    },
```